modules/mod_base/controllers/controller_postback.erl

```diff
@@ -28,6 +28,7 @@
 
 -include_lib("zotonic.hrl").
 
+-define(SIDEJOB_TIMEOUT, 1000).
```

Comment:
Yes, that is a difference between the previous method and this MR. Now the ack is only telling something about the transport, and basically says `ok` or `overload`.

For overload we also send back an ack for qos 0 messages. The client will not be able to resend the message but will back off for one second.